src/api/spec/controllers/staging/staging_projects_controller_spec.rb

```diff
@@ -86,29 +86,105 @@
         untracked_request.change_review_state(:accepted, by_group: staging_workflow.managers_group.title)
         bs_request.change_review_state(:accepted, by_group: staging_workflow.managers_group.title)
         logout
-        get :show, params: { staging_workflow_project: staging_workflow.project.name, staging_project_name: staging_project.name, format: :xml }
       end
 
-      it { expect(response).to have_http_status(:success) }
+      context 'with staged requests' do
```

Comment:
We should also test at least once with `request: 0`, `history: 0` or `status: 0`.

Reply:
After a short discussion, we decided that any value means "true". No value or a missing parameter means "false".
I'll adjust the conditions...